wave_calculator.gd

```diff
@@ -61,15 +92,19 @@ func _is_valid_score(score: int) -> bool:
 
 func _get_difficulty(wave_number: int) -> int:
 	# Diffuclty increases linearly, which is boring, should have some more interesting curve
-	return 5 + (wave_number * 4)
+	return 5 + log(wave_number + 1) + wave_number
```

Comment:
just trying something different - seems to work ok

Reply:
This gives an easier curve, lets try it out.

Reply:
removed `log` and went back to a more linear scale